client/src/components/SmallActionButton/SmallActionButton.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styles from "./SmallActionButton.module.css";
+import { PropTypes } from "prop-types";
+
+const SmallActionButton = ({ text, startEndClick }) => {
```

Comment:
I would perhaps rename the `startEndClick` prop to something a bit more descriptive such as `toggleButton` or something similar

Reply:
Rename `startEndClick` to `toggleButton`.